src/EditorFeatures/Core/InheritanceMargin/AbstractInheritanceMarginService.cs

```diff
@@ -76,11 +77,16 @@ internal abstract class AbstractInheritanceMarginService : IInheritanceMarginSer
             }
 
             var solution = project.Solution;
-            var serializedInheritanceMarginItems = await GetInheritanceMemberItemAsync(
-                solution,
-                project.Id,
-                symbolKeyAndLineNumbers,
-                cancellationToken).ConfigureAwait(false);
+            var serializedInheritanceMarginItems = ImmutableArray<SerializableInheritanceMarginItem>.Empty;
+            using (Logger.LogBlock(FunctionId.InheritanceMargin_TraverseInheritanceChain, cancellationToken, LogLevel.Information))
```

Comment:
This is the where the heavy work starts, I am interested in how long would this take. (Won't affect the main thread, only affect how soon will the user see the margin)